test/client_test.rb

```diff
@@ -109,23 +109,23 @@ def test_latency_on_dogstatsd_prefers_distribution_metric_type
 
   def test_latency_calls_block_even_when_not_sending_a_sample
     called = false
-    datagrams = @client.capture do
+    @client.capture do
       @client.latency('foo', sample_rate: 0) { called = true }
     end
-    assert_predicate datagrams, :empty?
```

Comment:
Note: because we use `capture`, the datagram will now always be captured no matter what the sample rate is.